src/styles/index.scss

```diff
@@ -1,3 +1,150 @@
-body {
+// Variables
+$primary-color: #007bff;
+$star-active-color: #ffc107;
+$star-inactive-color: #e4e5e9;
+$text-color: #333;
+$main-color: #060b35;
+$font-family: Roboto, sans-serif;
+$button-color: #00acdc;
+$price-code-color: #616070;
+$price-size: 12px;
+
+@mixin star($active: false) {
+  width: 20px;
+  height: 20px;
+  background: url('/src/images/star.svg') no-repeat center;
+  background-size: cover;
+
+  @if $active {
+    color: $star-active-color;
+  } @else {
+    color: $star-inactive-color;
+  }
+}
```

Comment:
1. reuse the logic from your `stars` homework. You should work with `stars` modifiers
2. move `stars` block styles to `stars.scss` file

Reply:
Not fixed